vtex/billing-app
# Feature/revoke

#### What is the purpose of this pull request?
Create a `revoke` feature to end contracts

#### What problem is this solving?
The impossibility of ending app contracts

#### How should this be manually tested?
1. Clone/pull this repo/branch
2. Install an app by buying it: `vtex install vtex.sms-provider -r vtex --verbose`
3. Revoke its contract by using this GraphQL mutation:
```
mutation {
  revoke(appName: "vtex.sms-provider") {
    revoked
  }
}
```
You should receive a string with the appId of the revoked contract

#### Types of changes
